moped-database/migrations/1709753345334_audit_fields_proj_files/up.sql

```diff
@@ -0,0 +1,20 @@
+ALTER TABLE moped_project_files RENAME COLUMN create_date TO created_at;
+ALTER TABLE moped_project_files RENAME COLUMN created_by TO created_by_user_id;
+
+ALTER TABLE moped_project_files ADD COLUMN updated_at TIMESTAMP WITH TIME ZONE;
```

Comment:
i suggest adding `set default now()` here

Reply:
or i guess if we prefer to have the default `updated_at` value for a new record be `null` then we should drop those defaults from the other tables. either way would make sense to me!

Reply:
thank you! I want to keep us in the same pattern, so I will add default now here